data/transactions/verify/txn.go

```diff
@@ -41,8 +41,9 @@ var logicErrTotal = metrics.MakeCounter(metrics.MetricName{Name: "algod_ledger_l
 // on a signed transaction.
 type Context struct {
 	Params
-	Group      []transactions.SignedTxn
-	GroupIndex int
+	Group          []transactions.SignedTxn
+	GroupIndex     int
+	MinTealVersion uint64
```

Comment:
I think this should go into `Params`, not `Context`. `Params` is used for caching verified transactions, whereas `Context` is not. See here.

https://github.com/algorand/go-algorand/blob/master/data/pools/transactionPool.go#L423

Reply:
Yeah, I think the group hash committing to the other transactions should take care of it, but it's trivial to move this to `Params`, so I just did that.